src/main/scala/esmeta/spec/util/Stringifier.scala

```diff
@@ -128,6 +128,8 @@ object Stringifier {
       case ButNot(base, cases) => app >> base >> " but not " >> cases
       case Empty               => app >> "[empty]"
       case NoLineTerminator    => app >> "[no LineTerminator here]"
+      case CodePoint(cp, desc) =>
+        app >> "<U+" >> cp >> (if (desc == "") "" else " " + desc) >> ">"
       case CodePointAbbr(abbr) => app >> "<" >> abbr >> ">"
```

Comment:
Isn't this still missing support for bare code point nonterminals described in the second bullet point at [grammarkdown Character Literals and Ranges](https://github.com/rbuckton/grammarkdown#character-literals-and-ranges) (e.g., `U+00A0` and `U+1D306`)?